doc/nrf/gs_modifying.rst

```diff
@@ -151,8 +221,39 @@ To locate a specific configuration option, use the filter (:guilabel:`Jump to` i
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
 .. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+   All changes to the :file:`.config` file are lost when you clean your build directory or you rebuild the project with the ``pristine`` target.
+   You can save the file to another location, but you must then manually copy it back to your build directory.
+   See Zephyr's :ref:`zephyr:application_rebuild` for more information.
+
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or one of the configuration file overlays.
+In these files, you can specify different values for configuration options that are defined by a library or board, and you can add configuration options that are specific to your application.
+
+See :ref:`zephyr:setting_configuration_values` in the Zephyr documentation for information on how to edit the :file:`prj.conf` file.
+
+.. tip::
+   When modifying configuration, especially when disabling one of the previously enabled options, check if the options required by the option you are disabling are still needed.
+
+When working with `Child images`_, you might need to permanently change their configuration.
+In such case, your application can provide overlay files for child images.
+
```

Comment:
We can't say that something can be done without linking them to docs about how to do it.

We need to add a line that links to docs about how to do this.

Reply:
The nordic docs are down. But I think it's documented here:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html

Reply:
I think we should remove those two lines:
- It's already described here: http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-6121/nrf/gs_modifying.html#child-images that child images can be configured, and that has a proper link
- The text is imprecise, because providing overlay files is just one option.
   It's also possible to completely replace the default config file for a child image, see: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#image-specific-variables
   `     |-- prj.conf (G)                    If present, use instead of (A)`

This again shows why we should be very careful about writing identical pieces of information in multiple places, cause the chance of one of the places being imprecise, or directly wrong is close to 100%.

Reply:
We have a mention about coonfiguring child images in the "changing configuration temporarily", so removing it from the "changing config permanently" makes the documentation unbalanced. I moved the link to replace the second sentence ("In such case...")

Reply:
> We have a mention about coonfiguring child images in the "changing configuration temporarily", 

No we don't.

We have a section about child images.
And after that child image section we have "changing configuration temporarily" which doesn't mention child images.

So why mention child images as part of "changing config permanently" when not mentioned under temporarily.

```
Child images
------------

The |NCS| build system allows the application project to become a root for the sub-applications known in the |NCS| as :ref:`child images <ug_multi_image>`.
The child images are system components that reside on the same hardware system.
They either cooperate with the application or the application depends on them.
Examples of child images are bootloaders, network cores, or security-related binaries.

Each child image is a separate application.
For information about how to configure child images, see :ref:`ug_multi_image_defining`.

Changing the configuration temporarily
======================================

When building your application, the different :file:`Kconfig` and :file:`*_defconfig` files and the :file:`prj.conf` file are merged together.
.....

```

Reply:
"Chaning the configuration temporarily" does mention child images:
```
To edit the file in SES, select :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK Project`.
If your application contains more than one image (see :ref:`ug_multi_image`), you must select the correct target.
To configure the parent image (the main application), select :guilabel:`menuconfig`.
The other options allow you to configure the child images.
```

Reply:
because in SES you get an interface like this:
![image](https://user-images.githubusercontent.com/9802583/145002935-a78f99eb-52ae-42ba-8fd2-d8c3da256797.png)

So this helps users to identify which image to configure, and as such has nothing to do with configuring of child images.
And as you see, it's only mentioned for SES, not in general (for `west` / `ninja`)

Reply:
Removed the two lines.